doc/user/content/_index.md

```diff
@@ -32,7 +32,12 @@ data freshness**.
 {{< linkbox icon="bulb" title="Key Concepts" >}}
 
 -   [Materialize overview](/overview/what-is-materialize/)
--   [Key Concepts](/concepts/)
+-   [Clusters](/concepts/clusters/)
```

Comment:
Since the concepts section is only a handful of pages right now, figured we might as well just list it out.